docs/self-managed/setup/deploy/amazon/amazon-eks/eks-helm.md

```diff
@@ -181,139 +223,424 @@ spec:
-This reveals a `client-id` and `client-secret` that can be used to connect to the Camunda 8 cluster.
+**Generating an M2M Token Using Identity:**
+
+You can generate an M2M token by following the steps outlined in the [Identity Getting Started Guide](/self-managed/identity/getting-started/install-identity.md), along with the [Incorporating Applications Documentation](/self-managed/identity/user-guide/additional-features/incorporate-applications.md).
+Below is an extract of the necessary instructions:
+
+<Tabs groupId="domain">
+  <TabItem value="with" label="With Domain" default>
+
+1. Open Identity in your browser at `https://${DOMAIN_NAME}/identity`. You will be redirected to Keycloak and prompted to log in with a username and password.
+2. Use `demo` as both the username and password.
+3. Click on "Add Application," select `M2M` as the type, and assign a name like "test."
+4. Select the newly created application, go to "Access to APIs" and click on the "Assign permissions" then select "Zeebe API" with "write" permission.
+5. Retrieve the `client-id` and `client-secret` values from the application details
+
+<!-- prevent error during compilation!-->
+  </TabItem>
+  
+  <TabItem value="without" label="Without Domain">
+
+This requires to port-forward the Identity and Keycloak to be able to connect to the cluster.
+
+```shell
+kubectl port-forward services/camunda-identity 8069:80 --namespace camunda
+kubectl port-forward services/camunda-keycloak 8070:80 --namespace camunda
+```
+
+1. Open Identity in your browser at `http://localhost:8069`. You will be redirected to Keycloak and prompted to log in with a username and password.
+2. Use `demo` as both the username and password.
+3. Click on "Add Application," select `M2M` as the type, and assign a name like "test."
+4. Select the newly created application, go to "Access to APIs" and click on the "Assign permissions" then select "Zeebe API" with "write" permission.
+5. Retrieve the `client-id` and `client-secret` values from the application details
+
+<!-- prevent error during compilation!-->
+  </TabItem>
+</Tabs>
+
+```shell
+export ZEEBE_CLIENT_ID='client-id' # retrieve the value from the identity page of your created m2m application
+export ZEEBE_CLIENT_SECRET='client-secret' # retrieve the value from the identity page of your created m2m application
+```
 
 <Tabs groupId="c8-connectivity">
-  <TabItem value="rest-api" label="REST API">
+  <TabItem value="rest-api" label="REST API" default>
 
 For a detailed guide on generating and using a token, please conduct the relevant documentation on [authenticating with the REST API](./../../../../../apis-tools/camunda-api-rest/camunda-api-rest-authentication.md?environment=self-managed).
 
 <Tabs groupId="domain">
-  <TabItem value="with" label="With Domain">
+  <TabItem value="with" label="With Domain" default>
 
 Export the following environment variables:
 
 ```shell
 export ZEEBE_ADDRESS=zeebe-rest.$DOMAIN_NAME
```

Comment:
didn't check 8.6 but at least for 8.7 (SNAPSHOT) the ingress is now combined for the rest endpoint.

```
export ZEEBE_ADDRESS=$DOMAIN_NAME/rest
```

Reply:
same is the case for 8.6